src/core/device.c

```diff
@@ -132,6 +291,23 @@ static int device_load(Unit *u) {
                         log_unit_debug_errno(u, r, "Failed to unescape name: %m");
         }
 
+        if (!verify_io_cost_supported(d)) {
+                log_debug_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "IO cost is not supported");
```

Comment:
no need to supply an error code here if you mark it as synthetic *and* don#t intend to return it... just use plain log_debug() in this case.